src/verify_ce_config.py

```diff
@@ -10,86 +10,135 @@
 
 
 def error(msg):
+    """Exit with error 'msg'
+    """
     sys.exit("ERROR: " + msg)
 
 
 def warn(msg):
+    """Print warning 'msg'
+    """
     print "WARNING: " + msg
 
 
+def parsed_route_names(entries_config):
+    """Return names of job routes that can be parsed as proper ClassAds
+    """
+    return [x['Name'] for x in classad.parseAds(entries_config)]
+
+
+def malformed_entries(entries_config):
+    """Find all unparseable router entries based on the raw JOB_ROUTER_ENTRIES configuration
+    """
+    unparsed_names = [x.replace('"', '')
+                      for x in re.findall(r'name\s*=\s*["\'](\w+)["\']',
```

Comment:
```suggestion
                      for x in re.findall(r'''name\s*=\s*["'](\w+)["']''',
```
The `r` causes the backslash to get in there literally, even when you're trying to quote the quote character.